pkg/storage/tsdb/tenant_deletion_mark.go

```diff
@@ -0,0 +1,39 @@
+package tsdb
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"path"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/thanos-io/thanos/pkg/objstore"
+)
+
+const TenantDeletionMarkFilename = "tenant-deleted.json"
```

Comment:
Two things:
1. Can we store it under the `marker/` prefix, within the tenant location? This way will work nicely with the bucket index too (we'll store the block deletion marks there).
2. What do you think to name it `tenant-deletion-mark.json` to keep naming consistent with the block deletion mark?

Reply:
Sure.